BCnEnc.Net/Decoder/BcDecoder.cs

```diff
@@ -35,34 +33,34 @@ public class BcDecoder
 		#region Async Api
 
 		/// <summary>
-		/// Decode raw encoded image asynchronously.
+		/// DecodeInternal raw encoded image asynchronously.
```

Comment:
Refactoring or intended?

Reply:
Refactoring by ReSharper. I will revert that.